redaxo/src/addons/phpmailer/lang/de_de.lang

```diff
@@ -66,7 +66,14 @@ phpmailer_archive = E-Mail-Archivierung
 phpmailer_archive_info = Archiv-Ordner: <span title="{0}">{1}</span>
 phpmailer_archive_delete = E-Mail-Archiv löschen
 phpmailer_archive_deleted = Das E-Mail-Archiv wurde gelöscht. 
-phpmailer_archive_delete_confirm = Soll das E-Mail-Archiv vollständig gelöscht werden? Dieser Vorgang kann nicht rückgängig gemacht werden.  
+phpmailer_archive_delete_confirm = Soll das E-Mail-Archiv vollständig gelöscht werden? Dieser Vorgang kann nicht rückgängig gemacht werden. 
+phpmailer_archivecron = Mailer-Archiv bereinigen
+phpmailer_archivecron_nothing_to_delete = Es wurde nichts zum Löschen gefunden
+phpmailer_archivecron_folder_not_found = Der Archiv-Ordner wurde nicht gefunden
+phpmailer_archivecron_label = Lösche Mails älter als 
+phpmailer_archivecron_days = Tage
+phpmailer_archivecron_delete = Anzahl gelöschter Objekte
+phpmailer_archivecron_notice = Datenschutzhinweis: Es sollte ein möglichst kurzer Zeitraum gewählt werden. 
```

Comment:
Im Screenshot steht noch 7 Tage gleich DSGVo konform. Da finde ich gut, dass du den Hinweis weggenommen hast, da wir keine rechtlichen Tipps geben sollten (Glatteis).
Selbst den jetzigen Hinweis weiß ich aber nicht, ob ich den einfügen würde. Tendenziell würde ich nur die Funktion anbieten, und man muss selber wissen, aus welchen Gründen man welchen Zeitraum benötigt. 
Aber vermutlich ist der Satz nicht problematisch, und kann daher meinetwegen sonst auch in der jetzigen Form drin bleiben.

Reply:
Ja war mir dann auch irgendwie zu viel. Habe das Notice nun komplett entfernt